pyproject.toml

```
@@ -12,10 +12,18 @@ maintainers = [
   {name = "The PyBOP Team"},
 ]
 description = "Python Battery Optimisation and Parameterisation"
-readme = "README.md"
+readme = {file = "README.md", content-type = "text/markdown"}
 license = { file = "LICENSE" }
-# https://pypi.org/classifiers/
-classifiers = []
+classifiers = [
```

```suggestion
classifiers = [
  "Development Status :: 3 - Alpha"
```

I guess this was the only one missing here. Please feel free to change it to Pre-Alpha or Beta if you feel like they will be more accurate.